packages/user-interface/src/contexts/AppContext.tsx

```diff
@@ -79,14 +79,10 @@ export const AppProvider = ({ children }: MessagesProviderProps) => {
           return;
         }
 
-        const style = await styleResponse.json();
-
-        Object.entries(style).forEach(([key, value]) => {
-          document.documentElement.style.setProperty(
-            key,
-            value as string | null,
-          );
-        });
+        const styleValue = await styleResponse.text();
+        const styleNode = document.createElement("style");
```

Comment:
This will not work without a nonce attribute if you have a CSP header that complies with the DigiD requirements.

Reply:
None of the utrecht styles that come from the den-haag component library have a nonce on it either, how is this one different?
<img width="590" height="444" alt="image" src="https://github.com/user-attachments/assets/e25605af-4a1c-451a-81f9-98046c2531d0" />

Reply:
Then what do you propose to make this work. Since a nonce is added by the webserver preferrably, that means we have a chicken-egg problem.